charts/sn-platform/templates/streamnative-console/streamnative-console-initialize.yaml

```diff
@@ -17,11 +17,16 @@
 # under the License.
 #
 
-{{- if .Values.initialize }}
 {{- if .Values.components.streamnative_console }}
 apiVersion: batch/v1
 kind: Job
 metadata:
+  annotations:
+    # This is what defines this resource as a hook. Without this line, the
+    # job is considered part of the release.
+    "helm.sh/hook": post-install,post-upgrade
```

Comment:
hmmmm, is `post-upgrade` a proper condition here? even we removed the .Values.initialize condition, the initialization job will only run once
if all the auth secrets are removed after a `helm upgrade` wouldn't pods fail to load secret when restarting?